.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       matrix:
         # Remember if you commit this, newly built images will replace those tags.
         # Prefer incrementing the version to unused one
-        tag: [18-android, 18-noble, 18-bookworm, 18-format, 18-mingw]
+        tag: [18-android, 18-noble, 18-bookworm, 18-format, 18-mingw, 18-emscripten]
```

Comment:
Please mark them as 19. I'd like to keep images already released immutable

Reply:
Got it, this has been done. Thanks